executor/aggfuncs/aggfunc_test.go

```diff
@@ -264,9 +264,9 @@ func rowMemDeltaGens(srcChk *chunk.Chunk, dataType *types.FieldType) (memDeltas
 	return memDeltas, nil
 }
 
-type multiArgsUpdateMemDeltaGens func(*chunk.Chunk, []*types.FieldType) (memDeltas []int64, err error)
+type multiArgsUpdateMemDeltaGens func(*chunk.Chunk, []*types.FieldType, []*util.ByItems) (memDeltas []int64, err error)
```

Comment:
can we use argvs[0] as `ByItems`? like `aggfuncTest`, so that we can remove `[]* util.ByItems`.

Reply:
If we have a table `t` and `GROUP_CONCAT( b,c order by c, b)` as follow, `argvs[0]` is  `column b`, ByItems include  `column b, c`. It seens that `[]* util.ByItems `must be used as a function parameter if we want to calculate `Datums` memory size.
`create table t(a bigint, b bigint, c bigint);`
`select GROUP_CONCAT( b,c order by c, b) from t ;`